src/components/forging/forging.pug

```diff
@@ -67,7 +67,7 @@ md-card.offline-hide
                     th(md-column) Total fee
                     th(md-column) Reward
                 tbody(md-body, infinite-scroll='$ctrl.loadMoreBlocks()', infinite-scroll-distance='1')
-                  tr(md-row, ng-repeat='block in $ctrl.blocks')
+                  tr(md-row, ng-repeat='block in $ctrl.blocks', data-open-explorer="{{block.id}}", data-url="'/block/'")
```

Comment:
I don't see the point of using two attributes. IMO it would be sufficient and simpler to have just one: `data-open-explorer="/block/{{block.id}}"`